.github/workflows/ci.yml

```
@@ -22,12 +26,16 @@ jobs:
         run: make check
       - name: distcheck
         run: make distcheck
-  test-ubuntu-oldest:
-    runs-on: ubuntu-16.04
+
+  test-centos:
+    runs-on: ubuntu-latest
+    container: centos
```

[The documentation](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idcontainer) suggests that this should instead be:

```suggestion
    container:
      image: centos
```

Yet it seemed to run OK so `¯\_(ツ)_/¯`